src/build.d

```diff
@@ -5,6 +5,12 @@ DMD builder
 Usage:
   ./build.d dmd
 
+detab, tolf, install targets - require the D Language Tools (detab.exe, tolf.exe)
+  https://github.com/dlang/tools.
+
+zip target - requires Info-ZIP or equivalent (zip32.exe)
+  http://www.info-zip.org/Zip.html#Downloads
```

Comment:
Remove this paragraph as this target is not defined by `dmd`.

Reply:
I added the zip target recently.

Reply:
Sorry for the noise - I missed that.